dev/integration_tests/web_e2e_tests/README.md

```diff
@@ -3,14 +3,16 @@
 These are web tests which are using `integration_test` (aka e2e) package. In order to run them locally, run `chromedriver`:
 
 ```
-./chromedriver --port=4444
+chromedriver --port=4444
 ```
 
+For more details on running/downloading chromedriver, use the [link](https://chromedriver.chromium.org/getting-started).
+
 Later use the following command:
 
 ```
 flutter drive --target=test_driver/text_editing_integration.dart -d web-server --browser-name=chrome
 ```
 
-For more details on running a driver test on web use the [link](https://github.com/flutter/flutter/wiki/Running-Flutter-Driver-tests-with-Web)
+For more details on running a Flutter Driver test on web use the [link](https://github.com/flutter/flutter/wiki/Running-Flutter-Driver-tests-with-Web)
```

Comment:
I need to bookmark and share this wiki link, users have asked me about "testing a package on the web", and I need to send them this.